package.json

```diff
@@ -54,5 +54,9 @@
 		"typescript": "^5.5.4",
 		"xo": "^0.59.3",
 		"yoctocolors": "^2.1.1"
+	},
+	"ava": {
+		"concurrency": 1,
+		"timeout": "60s"
```

Comment:
Taken from Execa. We are spawning hundreds of subprocesses at once, which can lead to timeouts or other issues. This helps prevent randomly failing tests.